packages/desktop-client/src/components/schedules/EditSchedule.js

```diff
@@ -85,7 +86,8 @@ export default function ScheduleDetails() {
 
   let { id, initialFields } = useParams();
   let adding = id == null;
-  let payees = useCachedPayees({ idKey: true });
```

Comment:
Not sure about the idKey here. It is used some places and not in others and there is no documentation on it. @MatissJanis is it related to the autocomplete work?

Reply:
It looks like it groups the payees by their ID for quick lookup (rather than putting them into an array)